### DISCUSSION TITLE
failed to create symbolic link '/home/lit/.lnd/lnd': Permission denied

### DISCUSSION THREAD

**Discussion by ghost** [lightninglabs/lightning-terminal#772] [Troubleshooting]
Running ubuntu 24.04 on arm board following raspibolt guide.
fails to create a symlink to the LND:
$ ln -s /data/lnd /home/lit/.lnd
ln: failed to create symbolic link '/home/lit/.lnd/lnd': Permission denied

I am not sure how to change permission to lit user properly.

**Comment by bitromortac** [comment]
This is a Linux problem, not related to lightning-terminal, you should log in as the correct user to create that folder (i.e. as the lit user, if that exists).

**Comment by ghost** [comment]
indeed, but I am logged as lit user
